filebeat/docs/modules/activemq.asciidoc

```diff
@@ -2,6 +2,8 @@
 This file is generated! See scripts/docs_collector.py
 ////
 
+:edit_url: https://github.com/elastic/beats/edit/main/x-pack/filebeat/module/activemq/_meta/docs.asciidoc
```

Comment:
Do we want to unset these at the end of the file? Even though they're going to be set again immediately at the beginning of the next file, it might be good to "clean up after ourselves" so to speak.

Reply:
Yup, I agree. Fixing right now.